tendrl/node_agent/alerts/alert_socket.py

```diff
@@ -0,0 +1,70 @@
+from _socket import error as _socket_error
```

Comment:
as in https://github.com/gevent/gevent/blob/master/src/gevent/server.py , gevent uses _socket which I found somewhere can throw the exception _socket._socket_error. So line 65 handles that exception. I haven't myself observed that happening though..